docs/src/examples/redis-clustering-aware.md

```diff
@@ -1,10 +1,10 @@
-# Redis Clustering with cluster aware client
```

Comment:
should the filename be updated to valkey as well?

Reply:
good catch, fixed